README.md

```diff
@@ -53,7 +53,7 @@ devtools::install_github(
 
 This package is not currently available on PyPi. To build the development version from source, clone this repo, then :
 
-```
+``` python
```

Comment:
I think there should be `shell` or maybe `bash`.

Reply:
Yes you are right. I am changed it

Reply:
Thanks for the comment @hpedrorodrigues! Yes, it should be `shell`. GitHub's markdown renderer, like many others, uses `highlight.js`. Here are all the supported languages, if you're curious: https://highlightjs.org/static/demo/